src/services/utilities/is-valid-coords-service.js

```
@@ -0,0 +1,66 @@
+        }
+        // { lat, lng }
+        else if (typeof param1 === 'object' && param1 !== null) {
+            param1 = param1.latitude ?? param1.lat;
+            param2 = param1.longitude ?? param1.lng ?? param1.lon ?? param1.long;
+            // Note: Above won't work because param1 is reassigned, so we need a temp object:
+        }
+
+        // Fix above logic for object case
+        if (typeof param1 === 'object' && param1 !== null) {
+            const parts = param1;
+            param1 = parts.latitude ?? parts.lat;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)